specification/search/data-plane/Azure.Search/preview/2025-08-01-preview/knowledgeagent.json

```diff
@@ -329,24 +352,45 @@
             }
           },
           "description": "The query details for the retrieval activity."
-        },
-        "queryTime": {
+        }
+      },
+      "description": "Represents a search index retrieval activity record."
+    },
+    "KnowledgeAgentAzureBlobActivityRecord": {
+      "type": "object",
+      "x-ms-discriminator-value": "azureBlob",
+      "allOf": [
+        {
+          "$ref": "#/definitions/KnowledgeAgentRetrievalActivityRecord"
+        }
+      ],
+      "properties": {
+        "searchTerm": {
           "type": "string",
-          "format": "date-time",
-          "description": "The query time for this retrieval activity."
-        },
-        "count": {
-          "type": "integer",
-          "format": "int32",
-          "description": "The count of documents retrieved."
+          "description": "The search term for the retrieval activity."
+        }
+      },
+      "description": "Represents a azure blob retrieval activity record."
+    },
+    "KnowledgeAgentWebActivityRecord": {
+      "type": "object",
+      "x-ms-discriminator-value": "web",
+      "allOf": [
+        {
+          "$ref": "#/definitions/KnowledgeAgentRetrievalActivityRecord"
+        }
+      ],
+      "properties": {
+        "searchTerm": {
+          "type": "string",
+          "description": "The search term for the retrieval activity."
         },
-        "elapsedMs": {
-          "type": "integer",
-          "format": "int32",
-          "description": "The elapsed time in milliseconds for the retrieval activity."
+        "freshness": {
+          "type": "string",
```

Comment:
Enum?

Reply:
no, this is a string of format "YYYY-MM-DD..YYYY-MM-DD"